css/interface.css

```diff
@@ -0,0 +1,676 @@
+html,
+body,
+#flooplan-app {
+  height: 100% !important;
+  overflow: hidden;
+}
+
+.fl-widget-provider {
+  padding: 15px;
+}
+
+main {
+  padding: 15px;
+  height: 100%;
+}
+
+#flooplan-app header {
+  margin-bottom: 0;
+}
+
+header .title-holder {
+  display: flex;
+}
+
+header .title-holder .go-back {
+  position: relative;
+  margin-right: 15px;
+  padding-left: 15px;
+  padding-right: 15px;
+  border-right: 1px solid rgba(51, 51, 51, 0.2);
+  line-height: 20px;
+  cursor: pointer;
+}
+
+header .title-holder .go-back .fa {
+  position: absolute;
+  top: 0;
+  left: 0;
+  font-size: 20px;
+  height: 20px;
+  line-height: 1;
+}
+
+header .title-holder p {
+  line-height: 20px;
+}
+
+.loading-message {
+  display: none;
+  padding-top: 25%;
+}
+
+.is-loading .loading-message {
+  display: block;
+}
+
+.is-loading .app-content {
+  opacity: 0;
+  pointer-events: none;
+}
+
+.spinner-holder {
+  text-align: center;
+}
+
+.app-content {
+  height: calc(100% - 27px);
+  overflow-y: auto;
+  overflow-x: hidden;
+  padding-top: 20px;
+  padding-bottom: 60px;
+}
+
+/* ACCORDION */
+.floor-panels-holder,
+.floor-markers-holder {
+  position: relative;
+  padding-bottom: 40px;
+  overflow: hidden;
+}
+
+.add-floor-panel,
+.add-floor-marker {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  text-align: center;
+  padding: 9px;
+  border: 1px solid #dcdae0;
+  color: #333;
+  opacity: 0.7;
+  border-radius: 4px;
+  background-color: transparent;
+  cursor: pointer;
+  z-index: 1;
+}
+
+.add-floor-panel:hover,
+.add-floor-marker:hover {
+  background-color: #f4f1f2;
+}
+
+.add-floor-panel:active,
+.add-floor-marker:active {
+  background-color: #f4f1f2;
+  -webkit-box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+  box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.125);
+}
+
+.floor-panels-holder .panel-group,
+.floor-markers-holder .panel-group {
+  display: block;
+  -webkit-user-select: none;
+  padding: 0 0 8px;
+  overflow-y: auto;
+  overflow-x: hidden;
+  margin: 0;
+}
+
+.floor-panels-holder.empty .panel-group,
+.floor-markers-holder.empty .panel-group {
+  display: none;
+}
+
+.panel {
+  opacity: 1;
+  -webkit-transition: opacity 100ms linear;
+}
+
+.panel.focus {
+  opacity: 0.7;
+}
+
+.panel-group .panel {
+  overflow: hidden;
+}
+
+.panel.focus .glyphicon-remove {
+  opacity: 0 !important;
+  -webkit-transition: opacity 50ms linear;
+}
+
+.panel-heading {
+  position: relative;
+  padding: 0;
+}
+
+.panel-heading:hover {
```

Comment:
You don't need `:hover` for cursors, because you'll only ever see the cursor when you hover :trollface:

Reply:
🤦‍♂️